include/multipass/timer.h

```diff
@@ -47,12 +47,13 @@ class Timer
     void pause();
     void resume();
     void stop();
+    TimerState current_state();
```

Comment:
```suggestion
    TimerState state();
```

Reply:
I committed in my branch to avoid the cla check failure.